data/kahet/kahet missions.txt

```diff
@@ -721,3 +721,1159 @@ event "pacified ka'sei"
+
+			label skipdialog
+			`	"On top of that, one of the discoveries that we have made is that based on our recordings, the Fetri'sei that attacked us was not waiting in the structure when we first entered the system; it was gathering materials from the asteroid belt together with the rest of its fellow Ka'sei initially, but changed path and moved inside the dome to await us as soon as we were noticed. If we could enter the system without being noticed, we could perhaps be able to explore the structure freely!"`
+
+			label proposal
+			choice
+				`	"What are you suggesting?"`
+			`	"Cloaking is the key to keep the Ka'sei from detecting and intercepting us. My idea is to equip a ship with that capability, a Starling for example, with enough ramscoops to remain cloaked indefinitely, as well as a Jump Drive to reach the system. It is extremely important to remain cloaked for the entire time to prevent the Ka'sei from engaging us, as otherwise it will go exactly like last time."`
+			`	After he finishes signing, he makes a few quick gestures toward the projector that you don't recognize, enlarging the hologram of the Builder settlement revealing what looks like a flight path drawn over the Builder ruins on the asteroid. "This is the path I was envisioning. You are going to land roughly where we did last time, carrying me and four other xenoarcheologists to expedite our work in the little time we can safely spend there. If you accept, I should be able to allocate <payment> to help you buy the necessary equipment for this mission. What do you think?"`
+			choice
+				`	"I'm in."`
+					goto in
+				`	"What if they detect us while inside the structure?"`
+				`	"Sorry, but I don't think it's a good idea to return there."`
+					decline
+			`	"It is a possibility, yes. It is also quite problematic because once we are in the airlock, it will be impossible to communicate through the structure to the outside, and vice versa. I think the best course of action would be, as soon as we are in, to place a scanner that controls the large airlocks on the upper side of the dome, the same ones the Ka'sei used last time; if I am right, it should give us around a quarter of an hour to get back in the airlock and escape safely."`
+			choice
+				`	"Alright, let's do it."`
+				`	"Sorry, but I don't think it's a good idea to return there."`
+					decline
+
+			label in
+			`	"Perfect. Remember to equip as many ramscoops as you can - they are going to be very useful this time."`
+				accept
```

Comment:
It isn't meant to handhold the player, but to ensure to prepare them for what will happen after the lab deployment (unbeknownst to Dusk or first last) - the missions thereafter will be harder and more frustrating without a sizeable amount of fuel or a good ramscooping capability. I've just used the excuse of the cloaked mission to allow a character to be able to warn the player

Reply:
The most fuel efficent roundtrip to Patir from the furthest remnant world is only 1400 fuel, and after 700 you can ramscoop very very quickly from Patir itself; during the mission chain instead 2200 fuel is needed, and this right after already doing a trip to Patir. While a few spots on the way allow for some ramscooping, if possible I'd like to leave at least the first mention, removing the second one a few missions after this

Reply:
If you want to add an incentive to install a ramscoop, then why not also add a condition where you have to stay in the system for a while, instead of just landing on the settlement?

Reply:
Eh, I don't think that flows very well, and I'm not a big fan of "stay in a system doing nothing for x time" mechanics. I'll just remove these, they're not strictly essential